File: Programming Basics with JS/T06 - While Loop/exercise/05. Coins.js
function coins(input) {
    let changeInCoins = Math.floor(Number(input[0]) * 100);
    let coins = 0;

    while (changeInCoins > 0) {
        if (changeInCoins >= 200) {
            changeInCoins -= 200;
            coins++;
        } else if (changeInCoins >= 100) {
            changeInCoins -= 100;
            coins++;
        } else if (changeInCoins >= 50) {
            changeInCoins -= 50;
            coins++;
        } else if (changeInCoins >= 20) {
            changeInCoins -= 20;
            coins++;
        } else if (changeInCoins >= 10) {
            changeInCoins -= 10;
            coins++;
        } else if (changeInCoins >= 5) {
            changeInCoins -= 5;
            coins++;
        } else if (changeInCoins >= 2) {
            changeInCoins -= 2;
            coins++;
        } else if (changeInCoins >= 1) {
            changeInCoins -= 1;
            coins++;
        }
    }
    console.log(coins);
}
// coins(['1.23']);
// coins(['2']);
// coins(['0.56']);
coins(['2.73']);

File: Programming Basics with Python/T06 - While Loop/exercise/05. Coins.py
change = float(input())
change_in_coins = int(change * 100)
coins = 0

while change_in_coins > 0:
    if change_in_coins - 200 >= 0:
        change_in_coins -= 200
        coins += 1
    elif change_in_coins - 100 >= 0:
        change_in_coins -= 100
        coins += 1
    elif change_in_coins - 50 >= 0:
        change_in_coins -= 50
        coins += 1
    elif change_in_coins - 20 >= 0:
        change_in_coins -= 20
        coins += 1
    elif change_in_coins - 10 >= 0:
        change_in_coins -= 10
        coins += 1
    elif change_in_coins - 5 >= 0:
        change_in_coins -= 5
        coins += 1
    elif change_in_coins - 2 >= 0:
        change_in_coins -= 2
        coins += 1
    else:
        change_in_coins -= 1
        coins += 1

print(coins)